rlw-examples/iis-monitoring.ps1

```diff
@@ -0,0 +1,5 @@
+# ---
+# RightScript Name: SYS IIS monitoring install
+# Description: Installs IIS specific monitoring.
+#
```

Comment:
This is not valid RightScript metadata, it needs more stuff including the terminating `...`:

```powershell
# ---
# RightScript Name: SYS IIS monitoring install
# Description: Installs IIS specific monitoring.
# Inputs: {}
# Attachments:
#   - attachments/iis-monitor.ps1
# ...
```